### PR TITLE
util: use uniform log format for util package

### PR DESCRIPTION
**Pull request by jackysp**
### What problem does this PR solve? 
Use uniform log format for util package

### Check List 

Tests 

 - Unit test

Code changes

 - Has exported function/method change

Related changes

 - Need to cherry-pick to the release branch

### CODE REVIEW

File: util/misc.go [hunk 1]
```diff
@@ -107,7 +109,7 @@ func SyntaxError(err error) error {
 	if err == nil {
 		return nil
 	}
-	log.Errorf("%+v", err)
+	log.Error(err.Error())
```
Comment (zimulala) [inline util/misc.go]: Do we add some information for this log?

File: util/printer/printer.go [hunk 2]
```diff
@@ -36,20 +37,20 @@ var (
...
+		zap.String("GoVersion", GoVersion),
+		zap.Bool("Race Enabled", israce.RaceEnabled),
+		zap.Bool("Check Table Before Drop", config.CheckTableBeforeDrop),
+		zap.String("TiKV Min Version", TiKVMinVersion))
 	configJSON, err := json.Marshal(config.GetGlobalConfig())
 	if err != nil {
 		panic(err)
 	}
-	log.Infof("Config: %s", configJSON)
+	log.Info("Loaded Config", zap.ByteString("config", configJSON))
```
Comment (zimulala) [inline util/printer/printer.go]: s/Loaded Config/loaded config

File: util/signal/signal_posix.go [hunk 3]
```diff
@@ -48,7 +50,7 @@ func SetupSignalHandler(shudownFunc func(bool)) {
 
 	go func() {
 		sig := <-closeSignalChan
-		log.Infof("Got signal [%s] to exit.", sig)
+		zaplog.Info("Got signal to exit", zap.String("signal", sig.String()))
```
Comment (zimulala) [inline util/signal/signal_posix.go]: s/Got/got

File: util/signal/signal_windows.go [hunk 4]
```diff
@@ -34,7 +35,7 @@ func SetupSignalHandler(shudownFunc func(bool)) {
 
 	go func() {
 		sig := <-closeSignalChan
-		log.Infof("Got signal [%s] to exit.", sig)
+		log.Info("Got signal to exit", zap.String("signal", sig.String()))
```
Comment (zimulala) [inline util/signal/signal_windows.go]: Ditto.

File: util/timeutil/time.go [hunk 5]
```diff
@@ -74,9 +74,9 @@ func InferSystemTZ() string {
 			if err2 == nil {
 				return name
 			}
-			log.Errorln(err2)
+			log.Error(err2.Error())
```
Comment (zimulala) [inline util/timeutil/time.go]: Do we need to add some information for this log?

File: util/timeutil/time.go [hunk 6]
```diff
@@ -74,9 +74,9 @@ func InferSystemTZ() string {
...
 		}
-		log.Errorln(err1)
+		log.Error(err1.Error())
```
Comment (zimulala) [inline util/timeutil/time.go]: Ditto.

File: util/misc.go (hunk 1 above)
Comment (jackysp) [reply util/misc.go]: I think no need for it. We could improve the message in parser repo.

**Comment by jackysp** [comment]
PTAL @zimulala

File: util/admin/admin.go [hunk 7]
```diff
@@ -127,7 +128,9 @@ func CancelJobs(txn kv.Transaction, ids []int64) ([]error, error) {
 		found := false
 		for j, job := range jobs {
 			if id != job.ID {
-				log.Debugf("the job ID %d that needs to be canceled isn't equal to current job ID %d", id, job.ID)
+				log.Debug("the job that needs to be canceled isn't equal to current job",
```
Comment (zz-jason) [inline util/admin/admin.go]: please use logutil

Comment (jackysp) [reply util/admin/admin.go]: Using a background context is not helpful either.

Comment (zz-jason) [reply util/admin/admin.go]: according to the discussing with @zimulala last few days ago, we should use logutil even if there is no valid context 😂

Comment (zimulala) [reply util/admin/admin.go]: Yes, we discussed it on engine channel, except for the people in their group, it seems that only @lysu and I are on that channel.
Sorry, I didn't inform you.

File: util/signal/signal_posix.go [hunk 8]
```diff
@@ -15,12 +15,14 @@
 package signal
 
 import (
+	"log"
```
Comment (XuHuaiyu) [inline util/signal/signal_posix.go]: useless import?

File: util/memory/action.go [hunk 9]
```diff
@@ -41,7 +41,7 @@ func (a *LogOnExceed) Action(t *Tracker) {
 	defer a.mutex.Unlock()
 	if !a.acted {
 		a.acted = true
-		log.Warnf(errMemExceedThreshold.GenWithStackByArgs(t.label, t.BytesConsumed(), t.bytesLimit, t.String()).Error())
+		log.Warn(errMemExceedThreshold.GenWithStackByArgs(t.label, t.BytesConsumed(), t.bytesLimit, t.String()).Error())
```
Comment (XuHuaiyu) [inline util/memory/action.go]: log.Warn(zap.Error(xxx)) ?

File: util/signal/signal_posix.go (hunk 8 above)
Comment (jackysp) [reply util/signal/signal_posix.go]: emm... If the logger package is down, sys log will work anyway.

File: util/misc.go (hunk 1 above)
Comment (lysu) [reply util/misc.go]: maybe better use `zap.Error`, our log analysis tool maybe need "error" key to identity error message part

File: util/memory/action.go (hunk 9 above)
Comment (lysu) [reply util/memory/action.go]: +1

File: util/signal/signal_posix.go [hunk 10]
```diff
@@ -48,7 +50,7 @@ func SetupSignalHandler(shudownFunc func(bool)) {
 
 	go func() {
 		sig := <-closeSignalChan
-		log.Infof("Got signal [%s] to exit.", sig)
+		zaplog.Info("got signal to exit", zap.String("signal", sig.String()))
```
Comment (lysu) [inline util/signal/signal_posix.go]: zap.Stringer

**Comment by zimulala** [comment]
PTAL @zz-jason @lysu @XuHuaiyu

**Pull request merged**